addon/models/goal.js

```diff
@@ -22,8 +25,13 @@ export default Activity.extend({
   description: attr('string'),
   progress: attr('string'),
   updateText: attr('string'),
-  product_owner: attr('number'),
-  head_design: attr('number'),
-  head_frontend: attr('number'),
-  head_backend: attr('number')
-});
\ No newline at end of file
+
+  contributers: hasMany('user'),
+
+  product_owner: belongsTo('user', { inverse: null }),
+  head_design: belongsTo('user', { inverse: null }),
+  head_frontend: belongsTo('user', { inverse: null }),
+  head_backend: belongsTo('user', { inverse: null }),
+
+  nr_volunteers: filterBy('user').length
```

Comment:
FilterBy should return an array with all users that match a certain criteria and are linked to this goal.
Because i don't add any criteria, it should be a "where true=true"-case, wich returns all users linked to this goal.
The length of that is the number of contributers